pom.xml

```diff
@@ -63,6 +63,9 @@
       <tag>HEAD</tag>
     </scm>
     <properties>
+        <jahia-deploy-on-site>all</jahia-deploy-on-site>
+        <jahia-module-type>system</jahia-module-type>
+        <require-capability>osgi.extender;filter:="(osgi.extender=org.jahia.bundles.blueprint.extender.config)"</require-capability>
```

Comment:
and this ?? I can't see any spring file here

Reply:
There is a Spring file in this PR, actually: https://github.com/Jahia/ckeditor/pull/13/files#diff-9d9ce5f04647499ccc7197916967bd7f
I allow to load the jQuery.js asset using old school org.jahia.ajax.gwt.helper.ModuleGWTResources
I didnt find any other way to inject easily a js into jahia root head html tag

Reply:
don't know how I missed the spring file that i commented .. monday morning